### PR TITLE
Scope revisions

### PR DESCRIPTION
**Pull request by noamross**
This is a revision of our scope guidelines, primarily clarifying language and providing examples. However, it includes some substantive changes, some of which are large but meant as starting points for conversation among @ropensci/editors. I hope to use this PR to facilitate this discussion and then resolve it in not too much time.

Substantive changes:

-   *drop the data visualization category*: This basically opens up to any htmlwidget package, and I don't see how to to deal with it it a reasonable way. It seems to have been there primarily for the legacy of **plotly**.
-   *"reproducibility" includes citation tools*: I believe **RefManageR** would fall under data munging, but citation is a core functionality for reproducibility that is specific to scientific workflows, and I think we should explicitly include it. Similarly I think **knitcitations** would fit into this mission.
-   *"reproducibility" does not include general R markdown tools*: I've tried to make a reasonable definition here, as this category can be very broad.
-   Explicit description of the text analysis pilot.
-   *add an explicit category for database wrappers*: We have a lot of these developed internally.

I think its fine that some legacy and current in-process packages would fall out of scope under changes, and I note this in the text.

This is kicked off from the useful comment from @statsmaths (please chime in with your thoughts on these changes - do they help?). Tagging @lmullen regarding the text analysis component.

### CODE REVIEW

**Comment by noamross** [comment]
Should I rename the section "Aims and Scope" as that parlance is familiar to journal people?

**Comment by maelle** [comment]
Thanks! :ok_hand: Regarding the general RMarkdown sentence I was first puzzled because of https://github.com/ropensci/ezknitr but then your last sentence about old packages clarifies it IMO.

Yes for renaming.

**Comment by lmullen** [comment]
I'm very glad that rOpenSci is willing to pilot a text analysis sub-speciality, and that the process is being formalized. 

My one suggestion pertains to a distinction that @noamross and I and perhaps others discussed on Slack. The idea was that text analysis packages could implement but not create new statistical methods. For example, wordVectors fits because it implements word2vec and can cite the Mikolov et al. paper. So rOpenSci is only undertaking the peer review of the package and not of word embedded models altogether.

Would that distinction be worth making in this text? Perhaps matters are sufficiently undecided that that distinction would be to precise.

**Comment by kbenoit** [comment]
I'm swamped with exams but just caught this thread. Where do I look to catch up with the subspecialty for text analysis packages? I'm very interested in this intriguing idea. Would be nice to be able to install the set using e.g. the tidyverse approach. The key will be in setting standards. Where is this discussion happening?

**Comment by noamross** [comment]
Whoops, meant to tag @statsmaths, not statsmath

**Comment by noamross** [comment]
@kbenoit See the `onboarding` Slack channel conversation starting January 25.  @lmullen Yes, I think that's correct. Will update to make this clear.

File: policies.md [hunk 1]
```diff
@@ -7,28 +7,84 @@
...
-Packages that fall into above categories but without any scientific applications are generally not accepted into rOpenSci. Standalone statistical tools (i.e. not associated with any of the categories above) are also not included in the suite. We encourage submitting those directly to CRAN as well as software journals such as JOSS, JSS, or the R journal
+Here are some types of packages we are unlikely to accept:
+
+-  Packages that wrap or implement statistical or machine learning methods.  We
+   are not organized so as to review the correctness of these methods. (But 
+   see "text analysis", above)
+-  Exploratory data analysis packages that visualize or summarize data. 
+-  General workflow or package development support packages
+
+We encourage submitting packages not accepted to rOpenSci those directly to CRAN,
```
Comment (sckott) [inline policies.md]: "We encourage submitting packages not accepted to rOpenSci directly to CRAN" ?

File: policies.md [hunk 2]
```diff
@@ -7,28 +7,84 @@
...
+to other R package development iniatives (e.g., [cloudyr](https://cloudyr.github.io/)),
+as well as software journals such as JOSS, JSS, or the R journal.
+
+Note that not all packages developed internally by rOpenSci or through our events
+or collaborations are in-scope for onboarding process.  As our scope has evolved
+over time, some previously packages would no longer be in-scope.
```
Comment (sckott) [inline policies.md]: "some packages we previously accepted would no longer be in-scope." ??

File: policies.md [hunk 3]
```diff
@@ -7,28 +7,84 @@
...
+
+* __data extraction__: Packages that aid in retrieving data from unstructured
+sources such as text, images and PDFs, as well as parsing scientific data types and
+outputs from scientific equipment. Statistical/ML libraries that enable data
+extraction are typically not included in this category.
+(Examples: [**tabulizer**](https://github.com/ropensci/onboarding/issues/42),
+[**robotstxt**](https://github.com/ropensci/onboarding/issues/25),
+[**genbankr**](https://github.com/ropensci/onboarding/issues/47))
+
+* __database access__: Bindings and wrappers generic databse APIs
```
Comment (sckott) [inline policies.md]: "Bindings and wrappers for generic databse APIs"

**Comment by sckott** [comment]
"Aims and scope" is good

I think we should state that:

- we may make exceptions under certain scenarios - though we strive to be consistent
- our policies/categories are a living document and change through time 

How do we think about the categories in `policies.md` jive with the categories used on the packages page https://ropensci.org/packages/ ?

**Comment by noamross** [comment]
Fixed typos @sckott found, addressed @sckott and @lmullen's comment.

These don't line up super well with the web site.  I'm not that concerned about this, as categorization for discovery doesn't serve the same role.  But if we get to serving that page via the RO API we might have package tags of some sort, which we can implement as tags in this repo. (`topic:` tags v `category:` tags, maybe)

**Comment by statsmaths** [comment]
I think these changes go a long way to making the requirements much more clear. Or, at least, it is clear that there requirements for text analysis packages are still very much a work in progress. 

On a related note, the website also has a section for image analysis. Is there plans to add a track for image processing as well? If there is a place where that discussion is happening I'd very much like to be involved.

**Comment by noamross** [comment]
@statsmaths There's not at the moment but we'll keep you in mind.  One thing we need to do is do a better job of distinguishing stuff developed in-house with stuff that's in-scope for onboarding. There's not 100% overlap currently.

**Comment by sckott** [comment]
True, i guess we don't need onboarding categories to match up with those on `/packages` page

**Comment by sckott** [comment]
> One thing we need to do is do a better job of distinguishing stuff developed in-house with stuff that's in-scope for onboarding. There's not 100% overlap currently.

we could have a badge on packages page (and elsewhere) that a pkg is community vs. staff (though some pkgs may not fit either of those perfectly if it’s a collaboration between staff and community)

**Comment by noamross** [comment]
I think what we need is not so much community v staff but peer-reviewed or not.  But before we get too far off topic, any other changes to the scope update, (or the EiC responsibilities?).  @karthik, haven't heard from you yet.  No opinions on dropping data viz or including citation tools?

**Comment by karthik** [comment]
@noamross Sorry, just getting around to reviewing this and the PR is merged. Thankfully this all makes sense to me. I also think that we need to have just one badge for peer-reviewed (and also make some policy/exception to grandfather all of rOpenSci's existing work that pre-dates onboarding). 

**Suggestions:**

> Reviewers: @githubname1 @githubname2 Due date: YYYY-MM-DD

Change that to 1 reviewer per line. That way if they are not starting the same day (e.g. you've assigned one reviewing but still seeking another), you can have separate due dates.

**Comment by njtierney** [comment]
Can I chime in about data visualisations?

There could still be scope for useful data visualisation tools that could be practical to specific fields - say for example certain extensions to ggplot, plotly, or leaflet, that might be very useful for weather plotting, or ecology. Recently, Carson put up plotdap for example.

Perhaps there could be a caveat that describes how the data visualisations need to be applied to specific examples? This is a tricky topic, I think!

**Pull request merged**